src/main/java/fi/fmi/avi/archiver/message/ProcessingResult.java

```diff
@@ -2,7 +2,13 @@
 
 public enum ProcessingResult {
 
-    OK(0), UNKNOWN_ICAO_CODE(1), MESSAGE_TIME_IN_FUTURE(2), MESSAGE_TOO_OLD(3);
+    OK(0),
+    UNKNOWN_ICAO_CODE(1),
+    MESSAGE_TIME_IN_FUTURE(2),
+    MESSAGE_TOO_OLD(3),
+    INVALID_MESSAGE_TYPE(4),
+    INVALID_MESSAGE_AERODROME(5),
+    INVALID_BULLETIN_LOCATION_INDICATOR(6);
```

Comment:
I got stuck in thinking about meaning of 'invalid'. These result codes are not used when the value in question is not invalid as itself, but within the context. Should the word be changed to something like 'unauthorized', 'forbidden', ...?

And should we use 'station' instead of 'aerodrome'?

Reply:
"Forbidden" seems best.

> And should we use 'station' instead of 'aerodrome'?

Yes.

Reply:
:+1: